### ISSUE TITLE
info from CLinvar

### ISSUE THREAD

**Issue by 4WGH** [Clinical-Genomics/scout#5133]
![image](https://github.com/user-attachments/assets/1a85b7d5-d54e-4980-9056-2db0d26fe946)
In pop up windonw with CLiVar information it would be very usefull if for variant with "Conflicting interpretation of pathogenicity" to see also the additinal text explaining te conflict as in the pic below 
![image](https://github.com/user-attachments/assets/b66df33a-2e8a-40b0-b081-cbb39bad738b)

if possible

**Comment by dnil** [comment]
Yep, we have requested this info to be added during annotation; it is available in ClinVar and VEP plugin! It should appear at the latest when we switch to nf-core/raredisease.

**Comment by dnil** [comment]
The syntax is like this: 
```
CLNSIG=Conflicting_classifications_of_pathogenicity|other;CLNSIGCONF=Uncertain_significance(1)|Benign(2)|Likely_benign(2)
```
